app/services/credit_notes/generate_service.rb

```diff
@@ -26,6 +26,8 @@ def call
     attr_reader :credit_note, :context
 
     def generate_pdf(credit_note)
+      I18n.locale = credit_note&.customer&.preferred_document_locale
```

Comment:
`credit_note` is always present here and the customer is always associated, no?

```suggestion
      I18n.locale = credit_note.customer.preferred_document_locale
```

Reply:
fixed